infra/api.py

```diff
@@ -100,6 +100,10 @@ def contributors(self):
         }
 
     def set_status(self, i, status):
+        if status == "denied-all":
+           for d in self.data:
+                if d['name'] == self.data[i]['name']:
+                    d['status'] = 'denied'
         self.data[i]['status'] = status
```

Comment:
Doesn't this then set the one you denied-all to have a weird new status?

Reply:
Oops, forgot a return statement. Added